controller/appcontroller.go

```diff
@@ -254,13 +255,27 @@ func (ctrl *ApplicationController) setAppManagedResources(a *appv1.Application,
 }
 
 // returns true of given resources exist in the namespace by default and not managed by the user
-func isKnownOrphanedResourceExclusion(key kube.ResourceKey) bool {
+func isKnownOrphanedResourceExclusion(key kube.ResourceKey, proj *appv1.AppProject) bool {
 	if key.Namespace == "default" && key.Group == "" && key.Kind == kube.ServiceKind && key.Name == "kubernetes" {
 		return true
 	}
 	if key.Group == "" && key.Kind == kube.ServiceAccountKind && key.Name == "default" {
 		return true
 	}
+
+	list := proj.Spec.OrphanedResourceWhitelist
```

Comment:
Please use `github.com/gobwas/glob` instead. Using glob implementation from `path/filepath` was a mistake and we should migrate to `github.com/gobwas/glob` everywhere.